examples/all-clusters-app/esp32/README.md

```diff
@@ -3,292 +3,22 @@
 A prototype application that demonstrates device commissioning and cluster
 control.
 
----
-
--   [CHIP ESP32 All Clusters Example](#chip-esp32-all-clusters-example)
-    -   [Supported Devices](#supported-devices)
-    -   [Building the Example Application](#building-the-example-application)
-    -   [Commissioning and cluster control](#commissioning-and-cluster-control)
-        -   [Setting up chip-tool](#setting-up-chip-tool)
-        -   [Commissioning over BLE](#commissioning-over-ble)
-        -   [Cluster control](#cluster-control)
-        -   [Flashing app using script](#flashing-app-using-script)
-        -   [Note](#note)
-    -   [Using the RPC console](#using-the-rpc-console)
-    -   [Device Tracing](#device-tracing)
+Refer [setup ESP-IDF and CHIP Environment](#setup-esp-idf-and-chip-environment)
```

Comment:
```suggestion
Refer to [setup ESP-IDF and CHIP Environment](#setup-esp-idf-and-chip-environment)
```

And similar in other places.

Reply:
Thanks Boris, i've changed this and others locally.